npe2/manifest/configuration._py

```diff
@@ -0,0 +1,262 @@
+# """
```

Comment:
what's the story here?  and what's `._py`?

Reply:
the coverage tool was complaining and I wasn't sure if we wanted to delete the files, keep them around or what. looking for recommendations.

Reply:
yeah, just leave this file untouched, comment it out in Contributions, and if coverage is the only thing that fails, it's fine with me (it will only happen once)

Reply:
reverted that commit